Containers/mastercontainer/start.sh

```diff
@@ -26,6 +26,12 @@ if [ "$EUID" != "0" ]; then
     exit 1
 fi
 
+# Check that the given command is not set
+if [ "$*" != "" ]; then
```

Comment:
Not completely sure if that works. It should check the options that are given to start.sh

Reply:
should work after googling what it does.

Reply:
but why this check? if you remove exec at the end of the file, the given cmd will be ignored anyway

Reply:
yes it will be ignored and then the user complains that the setting does not work even though they used the wrong command which is why I would rather add this check. However we could still remove the exec at the end of the file. WDYT?

Reply:
the exec at the end should be removed, but I still don't understand this check

Reply:
The initial report in https://github.com/nextcloud/all-in-one/discussions/3014 must have happened because they must have used something like 

```
sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest --env NEXTCLOUD_DATADIR="/mnt/ncdata/nextcloud"
```

If we now simply ignore this option without reporting back that it is incorrect, users will wonder why setting NEXTCLOUD_DATADIR does not work and will report a bug or open a thread. Reporting to them directly that this is wrong should hopefully work around this problem.

Reply:
I see then keep the check

Reply:
but exec at the end still need to be removed

Reply:
done